frontend/src/pages/LoginPage.js

```diff
@@ -80,67 +79,72 @@ class LoginPage extends Component{
 
     render() {
         return (
-            <div>
-                <div className="logo">
-                    <img
-                        src={`http://${process.env.REACT_APP_IP}/res/logo.png`}
-                        alt="Logo"
-                        width="340"
-                        onClick={this.onLogoClick}
-                    />
-                </div>
-                <div className="login-box-container">
-                    {this.state.redirect && <Redirect to={this.state.redirect} />}
-                    <p><font size ="5" color="#1DE28F"> Login </font></p>
-
-                    <form>
-                        <p>
+          <div>
+            <div className="logo">
+              <img
+                  src={`http://${process.env.REACT_APP_IP}/res/logo.png`}
+                  alt="Logo"
+                  onClick={this.onLogoClick}
+              />
+          </div>
+          <body className="align">
+            <div className="grid">
+                <form>
+                    <div className="form__field">
```

Comment:
change the class name. Why underscores, it doesn't fit with the rest, and is not descriptive enough